Client/qtTeamTalk/chattextlist.cpp

```diff
@@ -268,7 +268,7 @@ QStringList ChatTextList::allUrls(const QString &text) const
 
 QString ChatTextList::currentUrl(const QListWidgetItem* item) const
 {
-    QString text = item->text();
+    QString text = item->data(Qt::UserRole + 3).toString();

```

Comment:
Shouldn't here be a define of `Qt::UserRole + 3` item-data?

Reply:
UserRole + 3 is defined when we create items, e.g. line 195

Reply:
OK, but then make a define for each item-data. Otherwise the next developer will forget to update all places where it's used.

Reply:
Is f816daa what you want?